lldb/packages/Python/lldbsuite/test/tools/intelpt/intelpt_testcase.py

```diff
@@ -18,6 +18,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def skipIfCpuDoesNotSupportIntelPT(func):
```

Comment:
Maybe something slightly less verbose like `skipIfNoIntelPT`?
```suggestion
def skipIfNoIntelPT(func):
```